### PR TITLE
Fix_for_gettime.php

### PR DESCRIPTION
**Pull request by yogesh-m-2**

### CODE REVIEW

File: WEB-INF/lib/ttTimeHelper.class.php [hunk 1]
```diff
@@ -843,28 +843,6 @@ static function getAllRecords($user_id) {
 
     return $result;
   }
-
-  //return range of dates
-  static function getAllDateRecords($from_date,$to_date) {
```
Comment (naidu) [inline WEB-INF/lib/ttTimeHelper.class.php:848]: Why is this deleted?

File: api/deleteEntry.php [hunk 2]
```diff
@@ -44,25 +48,25 @@
...
-         else {
-           //if entry doesnt exist
-           $success_response = ['success' => false, 'error' => $record];
-           $response = json_encode($success_response);
-           print_r($response);
-              }
-                     }
+        $record = ttTimeClassHelper::deleteEntry($id);
+        if($record=="true"){
+          $success_response = ['success' => true, 'deleted data id' => $id];
```
Comment (naidu) [inline api/deleteEntry.php:53]: change `deleted data id` to `deleted id`.

File: api/gettime.php [hunk 3]
```diff
@@ -45,31 +49,27 @@
...
+        $response = json_encode($success_response);
+        print_r($response);
+
       }
       //if no values entered
       else
       {
         $to_date=$from_date;
-        $record = ttTimeHelper::getAllDateRecords($from_date,$to_date);
+        $record = ttTimeClassHelper::getAllDateRecords($from_date,$to_date);
```
Comment (naidu) [inline api/gettime.php:62]: Why call `ttTimeClassHelper::getAllDateRecords($from_date,$to_date);` twice?

File: WEB-INF/lib/ttTimeHelper.class.php (hunk 1 above)
Comment (yogesh-m-2) [reply WEB-INF/lib/ttTimeHelper.class.php:848]: i made a separate file for it and put it sir.
that function was made by me.

File: api/gettime.php (hunk 3 above)
Comment (yogesh-m-2) [reply api/gettime.php:62]: different case values are passed in the parameters, which depends on the if-else condition.
if the function is called only once globally. it might return wrong values when called inside the if-else condition.

Comment (naidu) [reply api/gettime.php:62]: I know.  When 

if($to_date=="''")
{
$to_date=$from_date;
}
and you dont need the else part

Comment (yogesh-m-2) [reply api/gettime.php]: edited now sir,
sorry about that.
